src/main/java/use_case/recipeCollection/RecipeCollectionInteractor.java

```diff
@@ -0,0 +1,33 @@
+package use_case.recipeCollection;
+
+import java.util.List;
+
+import entity.Recipe;
+
+/**
+ * The Recipe Collection interactor.
+ */
+public class RecipeCollectionInteractor implements RecipeCollectionInputBoundary {
+
+    private RecipeCollectionDataAccessInterface recipeCollectionDataAccessObject;
+    private RecipeCollectionOutputBoundary recipeCollectionPresenter;
+
+    public RecipeCollectionInteractor(RecipeCollectionDataAccessInterface recipeCollectionDataAccessInterface,
+                            RecipeCollectionOutputBoundary recipeCollectionOutputBoundary) {
+        this.recipeCollectionDataAccessObject = recipeCollectionDataAccessInterface;
+        this.recipeCollectionPresenter = recipeCollectionOutputBoundary;
+    }
+
+    /**
+     * Executes the Recipe Collection use case.
+     */
+    @Override
+    public void execute() {
+        final List<Recipe> recipeList = recipeCollectionDataAccessObject.getCurrentUserRecipeCollection();
+
+        final RecipeCollectionOutputData recipeCollectionOutputData = new RecipeCollectionOutputData(recipeList,
+                false);
```

Comment:
Looks a bit awkward with false sticking out but I understand its probably just barely over the recommended line limit. Maybe consider moving some more to the second line to make it look a little better.